Kitodo/src/main/java/org/kitodo/helper/Helper.java

```diff
@@ -584,4 +585,22 @@ public static String getLastMessage() {
     public static void setActiveMQReporting(Map<String, String> activeMQReporting) {
         Helper.activeMQReporting = activeMQReporting;
     }
+
+    /**
+     * Generate random string.
+     * 
+     * @param length
+     *            of random string to be created
+     * @return random string
+     */
+    public static String generateRandomString(int length) {
```

Comment:
Didn't we already have random generation somewhere!?

Reply:
We had in MockDatabase. It is removed from there in this PR and moved to Helper.

Reply:
I have seen this, I thought we had it somewhere for image files or something, but however...